.github/workflows/ci.yml

```diff
@@ -2,24 +2,48 @@ name: Build and test
 
 on:
   pull_request:
+  push:
+    branches:
+      - 10.*
+  schedule:
+    - cron: "0 0 */7 * *"
```

Comment:
Not super useful right now maybe, but if we reintroduce the auto-latest-release somehow it is good.